apparmor.d/profiles-s-z/sing-box

```diff
@@ -0,0 +1,37 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+# https://github.com/SagerNet/sing-box
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,local/}bin/sing-box
```

Comment:
Use `@{bin}/sing-box` (local program are purposely not confined by default profiles)